app/errors.py

```diff
@@ -0,0 +1,18 @@
+class VaccineError(Exception):
+    def __init__(self, message: str = "Default VaccineError message") -> None:
+        super().__init__(message)
+
+
+class NotVaccinatedError(VaccineError):
+    def __init__(self, message: str = "Default VaccineError message") -> None:

```

Comment:
Use a more descriptive default error message. For instance, 'The person is not vaccinated.' instead of 'Default VaccineError message'.